epub33/tts/index.html

```diff
@@ -234,12 +239,21 @@ <h3>The <code>ssml:alphabet</code> attribute</h3>
    &#8230;
    &lt;body>
    	&#8230;
-   	   &lt;p ssml:alphabet="ipa">&lt;span ssml:ph="ipʌb">EPUB&lt;/span> &#8230;&lt;/p>
+   	   &lt;p ssml:alphabet="ipa">&lt;span ssml:ph="ipʌb">EPUB&lt;/span> is an &#8230;&lt;/p>
    	&#8230;
    &lt;/body>
 &lt;/html></pre>
 				</aside>
 
+				<aside class="example">
+					<p>The following example shows a global declaration for the x-SAMPA alphabet on the root
+							<code>svg</code> element.</p>
+					<pre>&lt;svg &#8230; ssml:alphabet="x-sampa">
+   &lt;title>&lt;span ssml:ph="ipVb">EPUB&lt;/span> Adoption Chart&lt;/title>
+   &#8230;
+&lt;/svg></pre>
+				</aside>
+
```

Comment:
I think this should be `<tspan>` and not `<span>`

Reply:
Oops, sorry, I clicked the wrong line for the comment; I am talking about Example 4.

Reply:
Ya, obvious cut/paste mistake!